dataclients/kubernetes/definitions/testdata/errorwrapdata/routegroups.yaml

```diff
@@ -0,0 +1,21 @@
+items:
+  - apiVersion: zalando.org/v1
+    kind: RouteGroup
+    spec:
+        hosts:
+        - test.example.com
+        routes:
+        - filters:
+            - inlineContent("/foo")
+        - path: /foo
+  - apiVersion: zalando.org/v1
+    kind: RouteGroup
+    metadata:
+        name: rg1
+    spec:
+        hosts:
+        - test.example.com
+        routes:
+        - filters:
+            - inlineContent("/foo")
+        - path: /foo 
```

Comment:
Please add a newline. Makes sense to add a comment what's wrong with these routegroups.

Reply:
This should be resolved.